apis/risingwave/v1alpha1/risingwave_types.go

```diff
@@ -477,10 +493,62 @@ type RisingWaveObjectStorage struct {
 	AliyunOSS *RisingWaveObjectStorageAliyunOSS `json:"aliyunOSS,omitempty"`
 }
 
+// PersistentVolumeClaimPartialObjectMeta is the metadata for PVC templates.
+type PersistentVolumeClaimPartialObjectMeta struct {
```

Comment:
`kubectl explain pvc.metadata` lists these other fields as well

```
annotations
creationTimestamp
deletionGracePeriodSeconds
deletionTimestamp
finalizers
generateName
generation
labels
managedFields
name
namespace
ownerReferences
resourceVersion
selfLink
uid
```
Are we sure that we only need `Name`, `Labels`, `Annotations` and `Finalizers`?

What about the `Namespace` ? IMHO this may be important if we have tenants in different namespaces using PVCs with the same names.

Reply:
The `pvcTemplate` here is an equivalent of the `volumeClaimTemplates` in a `StatefulSet`. Most of the fields you listed above are useless.